HarmonyCore.CliTool/TUI/Views/EditSettingView.cs

```diff
@@ -101,16 +102,41 @@ public EditSettingView(string title, IHasNavigationResult navigationObject) : ba
                         X = 0,
                         Y = 2,
                         Width = Dim.Fill(),
-                        Height = Dim.Fill() - 1, //leave room for the buttons at the bottom
+                        Height = Dim.Fill() - 5, //leave room for the buttons at the bottom
                         RadioLabels = options.Select(obj => ustring.Make(obj.ToString())).ToArray(),
                         SelectedItem = options.IndexOf(oldValue.Value),
-                        
                     };
-                    Add(lbl, rg);
+                    var helpText = "Click an item to select it, or use arrow keys to move to an item," 
+                                 + "and press the spacebar to select it. Then click Ok or press Enter.";
+                    var affordance = new TextView() 
+                    {
+                        Text = helpText,
+                        X = 2,
+                        Y = 11,
+                        Width = Dim.Fill() - 4,
+                        Height = 3,
+                        ReadOnly = true,
+                        WordWrap = true,
+                    };
+                    Add(lbl, rg, affordance);
+                    ok.Enabled = false;
                     rg.SetFocus();
+                    rg.SelectedItemChanged += item =>
```

Comment:
is it possible to de-select an item or is it once selected, always selected?

Reply:
Once you select an item, you can deselect it by selecting another item (another radio button). Same as before. If you want to deselect whatever's selected without selecting something else, you Cancel.